packages/core/src/env.pipe.ts

```diff
@@ -4,8 +4,8 @@ import { Pipe, PipeTransform } from '@angular/core';
   name: 'env',
 })
 export class EnvPipe implements PipeTransform {
-  transform(variable: string): unknown {
-    const _variable = variable.replace('process.env.', '');
+  transform(variable: string): string {
+    const _variable = variable.replace(/^process.env./i, '');
```

Comment:
Sorry @AbderrahimSoubaiElidrissi just noticed the `/i`, can we let it case sensitive `/^process.env./` for consistency with TypeScript usage? Thanks a lot.

Reply:
@chihab Sure `1ff29c7`